sdk/trace/src/testIncubating/java/io/opentelemetry/sdk/trace/TracerConfigTest.java

```diff
@@ -11,6 +11,7 @@
 import static io.opentelemetry.sdk.trace.internal.TracerConfig.defaultConfig;
 import static io.opentelemetry.sdk.trace.internal.TracerConfig.disabled;
 import static io.opentelemetry.sdk.trace.internal.TracerConfig.enabled;
+import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
```suggestion
```

Reply:
Why does the build pass despite this unused import? 🤔